doc/developer/release-checklist.md

```diff
@@ -120,6 +120,26 @@ All of these can be run in parallel.
     ./dc.sh backup
     ```
 
+- [ ] Run the perf-kinesis load test on the tag.
+
+  - [ ] Spin up a fresh VM and start the load test. To start the load test, ssh
+    into the VM and run:
+    ```shell script
+    cd materialize
+    bin/mzcompose -f test/performance/perf-kinesis/mzcompose up -d
```

Comment:
It is a bit more idiomatic to use the mzcompose shortcut script:

```
cd materialize/test/performance/perf-kinesis
./mzcompose up -d
```

(I think perhaps that doesn't exist for perf-kinesis; you can generate it with `bin/mzcompose gen-shortcuts`.)